### PR TITLE
Fix for the 'You don't need to. . .' bug in crossing-repair

### PR DESCRIPTION
**Pull request by b-Y-d**
In case of hand-exploding backlash. For crossing-trainer.lic
I am new to this githubbery

### CODE REVIEW

**Comment by rpherbig** [comment]
`Sorcery.lic` *should* already be doing this: https://github.com/rpherbig/dr-scripts/blob/master/sorcery.lic#L42

Is it not working?

**Comment by b-Y-d** [comment]
I think it missed it at least once, found my bard logged via afk.lic sitting in my training room. (Not in the sorcery room). I'd have to check the logs to see what happened exactly.

**Comment by rpherbig** [comment]
If you could check your logs, that would be great

File: crossing-repair.lic [hunk 1]
```diff
@@ -32,11 +32,13 @@ class CrossingRepair
       fput('swap') unless right_hand
       command = "give #{repairer}"
     else
-      return unless @equipment_manager.get_item?(item)
+      unless item.name.split.last == left_hand.split.last || right_hand.split.last
```
Comment (rcuhljr) [inline crossing-repair.lic]: This won't work like you want, == has a higher operator precedence than ||.  [Precedence table](https://ruby-doc.org/core-2.2.0/doc/syntax/precedence_rdoc.html)

File: crossing-repair.lic [hunk 2]
```diff
@@ -32,11 +32,13 @@ class CrossingRepair
       fput('swap') unless right_hand
       command = "give #{repairer}"
     else
-      return unless @equipment_manager.get_item?(item)
+      unless item.name.split.last == (left_hand.split.last || right_hand.split.last)
```
Comment (rpherbig) [inline crossing-repair.lic]: The `split` here worries me. Shouldn't they match without splitting?

**Comment by b-Y-d** [comment]
**crossing-training.lic:** Removed the change/addition of running safe-room after sorcery to. Looks like I had safe-room paused for some reason when sorcery tried to run it that one time.

**crossing-repair.lic:** removed the if block with the splits, combined the two repeat cases.

**Pull request merged**